types/context.go

```diff
@@ -257,7 +257,7 @@ type cloner interface {
 	Clone() interface{} // deep copy
 }
 
-// XXX add description
+// TODO add description
```

Comment:
same

Reply:
Haven't diven in - I'm not actually positive on the functionality of Op at this point so I don't want to add the description now... however I'll note that XXX should not be used for non-critical areas of the code so I've renamed to TODO. Eventually when/if I dive into the `Context` type I will add this TODO, for now, I'm not focused on this code however.

Reply:
Is the `Context` refactor work something we could hand off to @sabau ?